src/encoded/audit/file.py

```diff
@@ -5,6 +5,27 @@
 
 from .item import STATUS_LEVEL
 
+def check_award_condition(value, awards):
+    return value.get('award', None) and value.get('award')['rfa'] in awards
+
+def audit_file_processed_step_run(value, system):
+    if value['status'] in ['replaced',
+                           'deleted',
+                           'revoked',
+                           'uploading',
+                           'content error',
+                           'upload failed']:
+        return
+    if value['output_category'] in ['raw data',
+                                    'reference']:
+        return
+    if check_award_condition(value.get('dataset'), ['ENCODE3', 'ENCODE4']):
```

Comment:
Isn't `value.get('dataset')` here just the accession of the experiment? How would passing this into check_award_condition work?

Reply:
'dataset' is embedded in file audits (see line 356) , it is not just an accession